src/daemon/daemon.cpp

```diff
@@ -1778,7 +1768,6 @@ void mp::Daemon::on_stop()
 
 void mp::Daemon::on_suspend()
```

Comment:
I suppose this override could be removed from the daemon entirely?

Reply:
It could, but I think that is out of scope for this PR.  The other `on_foo()` methods are the same as well.

Reply:
I understand what you say in regard to the other dangling `on_foo` methods, but this one *becomes* dangling in this PR if not removed. Personally I would have nothing against cleaning other `on_foo` methods here (that is justified by consistency with other PR changes IMO), but that's a separate question.

Reply:
I understand what you are saying about it becoming dangling, but I'm not keen on messing with the VMStatusMonitor class with this PR.